initramfs.sh

```diff
@@ -612,6 +612,7 @@ main() {
     fi
 
     if [[ $DO_ALL == 1 ]] || [[ $DO_TOOLS == 1 ]]; then
+        build_modules
```

Comment:
This should be on the `if [[ $DO_ALL == 1 ]] || [[ $DO_CORES == 1 ]]; then` block